babel/src/main/java/org/apache/calcite/sql/babel/SqlColumnAttributeDateFormat.java

```diff
@@ -42,9 +41,7 @@ public SqlColumnAttributeDateFormat(SqlParserPos pos, SqlNode formatString) {
 
   @Override public void unparse(final SqlWriter writer, final int leftPrec,
       final int rightPrec) {
-    if (formatString != null) {
-      writer.keyword("FORMAT");
-      formatString.unparse(writer, 0, 0);
-    }
+    writer.keyword("FORMAT");
```

Comment:
Suggest adding the null check back, but reversing the order such that you return early if it is null.

Reply:
Actually, now when I look at the code. It seems like formatString could not be null, since we assigned the value from StringLiteraL() to it in ParserImpls.ftl

Reply:
Okay, will add guard in the next commit.

Reply:
Can't another approach be to to this.formatString = Object.requireNotNull(formatString)? I'm not 100% that is the correct call but something like that.

Reply:
Guard added back in https://github.com/googleinterns/calcite/pull/103/commits/b63554f4f67218e8e176cea3d14790c698612c18

Reply:
Yes, Objects.requireNonNull(...) would be a valid alternative (and possibly better).